src/sema/ast.rs

```diff
@@ -1072,6 +1072,12 @@ pub enum Expression {
         array: Box<Expression>,
         index: Box<Expression>,
     },
+    NamedSubscript {
```

Comment:
"Subscript" makes me think of the subscript operator `[]`. I don't think this is a good name.

What do you think of calling it either `Member` or `NamedMember`?

Reply:
Changed it to `NamedMember`